### DISCUSSION TITLE
resetting advancements

### DISCUSSION THREAD

**Discussion by ghost** [TheSummerGrinch/MCManHunt#12] [Ideas]
it would be nice if there was a toggle-able feature to reset the advancements of participating players for the duration of the current match. as it is currently, if a player got an advancement in a previous match they will not receive it in the next match (if an admin did not manually reset their advancements).

**Comment by TheSummerGrinch** [comment]
Great idea! I've already started implementation as this was recently suggested in the Discord server as well, so it should be done this week!

This will also feature a slight overhaul of the config.yml to make it more intuitive!

I'll make it an Issue to make sure I don't forget.

Thanks for your suggestion, and if you have more suggestions feel free to contact me, or make an Issue!
